src/index.tsx

```diff
@@ -97,13 +121,14 @@ function LazyHydrate(props: Props) {
     }
 
     if (whenIdle) {
-      // @ts-ignore
-      if (requestIdleCallback) {
-        // @ts-ignore
-        const idleCallbackId = requestIdleCallback(hydrate, { timeout: 500 });
+      if (window.requestIdleCallback) {
+        const idleCallbackId = window.requestIdleCallback(hydrate, {
+          timeout: 500
+        });
         cleanupFns.push(() => {
-          // @ts-ignore
-          cancelIdleCallback(idleCallbackId);
+          if (window.cancelIdleCallback) {
```

Comment:
no if condition, you should have the polyfill

Reply:
This has been updated in #24 to check if (typeof window !== "undefined") first